server/listeners/sqsDomainEventsListener.ts

```diff
@@ -1,12 +1,17 @@
-import AWS from 'aws-sdk'
+import { SQSClient } from '@aws-sdk/client-sqs'
 import { Consumer } from 'sqs-consumer'
 import config from '../config'
 import logger from '../../logger'
 import buildEventHandler from './eventHandlers/domainEvents'
 import { Services } from '../services'
 
-const sqs = new AWS.SQS({
+const sqs = new SQSClient({
   region: 'eu-west-2',
+  endpoint: config.sqs.endpoint,
+  credentials: {
+    accessKeyId: 'foo',
+    secretAccessKey: 'bar',
+  },
```

Comment:
this shouldn't have credentials defined - these are picked up from environmental variables when running locally (from features.env in integration tests or .env when running locally), or via [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) when running in kubernetes.